### PR TITLE
[Term entry] Kotlin math_method: acosh()

### PR DESCRIPTION
**Pull request by HKABIG**
created asin.md in accordance to the guidelines provided

### Description

created docs/content/kotlin/concepts/math-methods/terms/acosh/acosh.md entry for https://github.com/Codecademy/docs/issues/3513

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by HKABIG** [comment]
@SSwiniarski please review this changes

File: content/kotlin/concepts/math-methods/terms/acosh/acosh.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Syntax
+
+```pseudo
+acosh(x)
+```
+
+The function accepts one mandatory parameter `x`, which should be a numeric value greater than or equal to `1`. The argument can be of `Double` datatype.
+
+It returns the hyperbolic arccosine of the argument, which represents a real number.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/acosh/acosh.md:27]:
```suggestion
The function accepts one mandatory parameter `x`, which should be a numeric value greater than or equal to `1`. The argument can be of `Double` and `Float` datatype. 

> **Note**: It returns the hyperbolic arccosine of the argument, which represents a real number.
```
We can take float datatype also. [ref](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.math/acosh.html)

File: content/kotlin/concepts/math-methods/terms/acosh/acosh.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+```kotlin
+import kotlin.math.acosh
+fun main() {
+    // Calculate the hyperbolic arccosine of 2.0
+    val x = 2.0
+    val result = acosh(x)
+
+    // Print the result
+    println(result)
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/acosh/acosh.md:41]:
```suggestion
import kotlin.math.acosh
fun main() {
    val x = 2.0
    val result = acosh(x)

    println(result)
```
No need for code comments here

File: content/kotlin/concepts/math-methods/terms/acosh/acosh.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+Title: 'acosh()'
+Description: 'Returns the hyperbolic arccosine of a numeric argument.'
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/acosh/acosh.md:3]:
```suggestion
Description: 'Returns the inverse hyperbolic cosine of a numeric argument.'
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side for now. Please commit it so we can move forward.

**Comment by HKABIG** [comment]
@dakshdeepHERE I have commited your suggestions

**Comment by dakshdeepHERE** [comment]
@HKABIG That was quick 😄, let me look through the entry again. 🚀

**Comment by dakshdeepHERE** [comment]
@HKABIG LGTM for 2nd review. Thanks for commits.🚀

**Review by dakshdeepHERE — APPROVED** [review]

**Review by SSwiniarski — APPROVED** [review]
@HKABIG I did some minor edits and now it looks good to merge.

**Pull request merged**